samples/dnn/DaSiamRPN_visual_tracker.py

```diff
@@ -0,0 +1,330 @@
+"""
+DaSiamRPN tracker.
+Original paper: https://arxiv.org/abs/1808.06048
+Link to onnx models: https://drive.google.com/drive/folders/1UuwZsgoOVJfwHdE7rS8dtG4UvYfr8OGi?usp=sharing
+Link to original repo: https://github.com/foolwood/DaSiamRPN
+"""
+
+import numpy as np
+import cv2 as cv
+import glob
+import argparse
+
+#function for cropping image
+def get_subwindow_tracking(im, pos, model_sz, original_sz, avg_chans):
+    if isinstance(pos, float):
+        pos = [pos, pos]
```

Comment:
Is that line reachable?

Reply:
I think no. It may be a bug - pos is nd.array. I think here should be check for type of pos[0], pos[1], not whole array.